client/lib/plans/features-list.js

```diff
@@ -285,7 +285,7 @@ export const FEATURES_LIST = {
 	[ constants.FEATURE_UNLIMITED_STORAGE ]: {
 		getSlug: () => constants.FEATURE_UNLIMITED_STORAGE,
 		getTitle: () =>
-			i18n.translate( '{{strong}}200 GB{{/strong}} Storage Space', {
+			i18n.translate( '{{strong}}200 GB{{/strong}} storage space', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.